code/game/jobs/job/science.dm

```diff
@@ -212,19 +212,26 @@
 	uniform = /obj/item/clothing/under/rank/scientist/botany
 
 /datum/job/intern_sci
-	title = "Lab Assistant"
+	title = "Research Understudy"
```

Comment:
```suggestion
	title = "Research Intern"
```

Reply:
I opted for the name "Understudy" because "Intern" didn't give the impression of someone who, in a reasonable length of playable time, is about to gain their PhD (As required to become a full scientist). If the consistency is really needed, I am happy to change it.